### ISSUE TITLE
read access on bootstrap css file

### ISSUE THREAD

**Issue by robjhyndman** [quarto-dev/quarto-cli#11532]
### Bug description

Under quarto 1.6.37, when generating my website (robjhyndman.com), the `bootstrap-<string of characters>.min.css` file that is created (within the `_site/site_libs/bootstrap` folder) has permissions `-rw-------` instead of `-rw-r--r--`. This means the site renders fine locally, but when uploaded to a server, it is broken (that file returns a 403 error).

I have fixed it using `chmod +r` on the server, but this has to be done every time I regenerate the site.

The problem didn't occur using quarto 1.5.57.

### Quarto check output

```
Quarto 1.6.37
[✓] Checking environment information...
      Quarto cache location: /home/hyndman/.cache/quarto
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.4.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.46.3: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.6.37
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /usr/bin
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.3
      Path: /usr/bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

[✓] Checking R installation...........OK
      Version: 4.4.2
      Path: /usr/lib/R
      LibPaths:
        - /home/hyndman/R/x86_64-pc-linux-gnu-library/4.4
        - /usr/local/lib/R/site-library
        - /usr/lib/R/site-library
        - /usr/lib/R/library
      knitr: 1.49
      rmarkdown: 2.29

[✓] Checking Knitr engine render......OK
```

**Comment by cscheid** [comment]
> We'll fix that, and probably backport to 1.6 in our next patch release. To be confirmed with @cscheid.

This is definitely worth a backport and early patch release.

**Comment by cderv** [comment]
This was fixed in another PR 
- https://github.com/quarto-dev/quarto-cli/pull/11539

and it has been backported to v1.6. We'll do a patch release soon.
